src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const row = rows[i];
+
+  const clonedCell = row.cells[1].cloneNode(true);
+
+  row.insertBefore(clonedCell, row.cells[row.cells.length - 1]);
```

Comment:
This line will insert the cloned cell before the last cell of the row. If you want to append the cloned cell at the end of the row, you can use `row.appendChild(clonedCell)` instead.